lib/validator.js

```diff
@@ -552,21 +544,37 @@ Specberus.prototype.getChartersAndPolicy = async function () {
                     docDate <= new Date(groupCharter.end)
                 ) {
                     chartersData.push(groupCharter);
-                    charters.push(groupCharter.uri);
-                    patentPolicy.push(groupCharter['patent-policy']);
                 }
             });
         });
     }
 
     this.chartersData = chartersData;
+    return chartersData;
+};
+
+Specberus.prototype.getCharters = async function () {
+    if (undefined !== this.charters) {
+        return this.charters;
+    }
+    const chartersData = await this.getChartersData();
+    const charters = [];
+    chartersData.every(charterData => charters.push(charterData.uri));
     this.charters = charters;
-    this.patentPolicy = patentPolicy;
-    return {
-        chartersData,
-        patentPolicy,
-        charters,
-    };
+    return charters;
+};
+
+Specberus.prototype.getPatentPolicies = async function () {
+    if (undefined !== this.patentPolicies) {
+        return this.patentPolicies;
+    }
+    const chartersData = await this.getChartersData();
+    const patentPolicies = [];
+    chartersData.every(charterData =>
```

Comment:
you are mistaking `every` with `forEach`. [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) should be use to test all elements of the array.
```suggestion
    chartersData.forEach(charterData =>
```